.github/markdown-link-check-config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Inspired by: https://github.com/lmc-eu/cookie-consent-manager/pull/53

Reply:
Some links in `web-react/CHANGELOG.md` return `404`.

How you wanna solve these scenarios? Via [disable/enable comments](https://github.com/DavidAnson/markdownlint#tags)? @literat

Reply:
`spirit-web` -> `spirit-web-react` - separate commit.